packages/composites/proof-of-reserves/src/reduce.ts

```diff
@@ -30,6 +31,8 @@ export const runReduceAdapter = async (
   switch (indexer) {
     case bitcoinJsonRpc.NAME:
       return returnParsedUnits(input.jobRunID, input.data.result, 8)
+    case bitcoinPorIndexer.NAME:
+      return returnParsedUnits(input.jobRunID, input.data.result, 8)
```

Comment:
Wouldn't be too strange given the `lotus` and `adaBalance` combination that follows
```suggestion
    case bitcoinJsonRpc.NAME:
    case bitcoinPorIndexer.NAME:
      return returnParsedUnits(input.jobRunID, input.data.result, 8)
```